terraform/tests/sanity_check.tf

```diff
@@ -0,0 +1,34 @@
+module "mongos" {
+  source   = "../"
+  app_name = var.mongos_name
+  model    = var.model_name
+  units    = "1"
+  channel  = "6/edge"
+}
+
+
+resource "juju_integration" "data-integrator_mongos-integration" {
+  model = var.model_name
+
+  application {
+    name = juju_application.data-integrator.name
+  }
+  application {
+    name = var.mongos_name
+  }
+  depends_on = [
+    juju_application.data-integrator,
+    module.mongos
+  ]
+
+}
+
+
+resource "null_resource" "juju_wait_deployment" {
```

Comment:
In another PR, we do not need this as part of the TF modules. The ran command can just be part of the CI.